VMEncryption/main/BekUtil.py

```diff
@@ -82,6 +83,8 @@ def get_bek_passphrase_file(self, encryption_config):
 
             for file in os.listdir(self.bek_filesystem_mount_point):
                 if bek_filename in file:
+                    # Make sure LinuxPassPhraseFileName exist for Resource and Nvme disks 
```

Comment:
Should avoid self copy or unnecessary overwriting:
maybe add:
if bekfilename != file and not os.file.exists(bekfilename):